src/store/lessonsSlice.ts

```diff
@@ -6,6 +6,7 @@ export type LessonsSlice = {
 };
 
 export type Lesson = {
+  title: string;
```

Comment:
I know this isn't part of your PR, but thoughts on renaming this to Playlist instead of Lesson to be more in sync with Playlist card?